### PR TITLE
Mail functionality cleanup

### PR DESCRIPTION
**Pull request by skamoen**
The code was old and bad, with terrible error handling.

This PR mostly cleans up the error handling, but the actual service and controller should be fixed to be used with a nice admin panel. 

BONUS:
capture

### CODE REVIEW

File: src/main/java/ch/wisv/areafiftylan/service/MailServiceImpl.java [hunk 1]
```diff
@@ -62,39 +72,40 @@ private String prepareHtmlContent(String name, String message) {
...
+    public void sendTemplateMailToUser(User user, MailDTO mailDTO) {
+        sendMail(user.getEmail(), user.getProfile().getFirstName(), mailDTO.getSubject(), mailDTO.getMessage());
     }
 
     @Override
-    public void sendVerificationmail(User user, String url) throws MessagingException {
-        String message = "Please click on the following link to complete your registration: " + url;
-        sendMail(user.getEmail(), user.getUsername(), null, "Confirm your registration", message);
+    public void sendVerificationmail(User user, String url) {
+        String message = "Please click on the following link to complete your registration: <a href=\"" +
```
Comment (TimvdLippe) [inline src/main/java/ch/wisv/areafiftylan/service/MailServiceImpl.java]: The `<a` is not closed with `</a>`. Intended? Same for the method below.

File: src/main/java/ch/wisv/areafiftylan/service/TeamServiceImpl.java [hunk 2]
```diff
@@ -118,12 +117,7 @@ public TeamInviteToken inviteMember(Long teamId, String username) {
             TeamInviteToken inviteToken = new TeamInviteToken(user, team);
             teamInviteTokenRepository.save(inviteToken);
 
-            try {
-                mailService.sendTeamInviteMail(user, team.getTeamName(), team.getCaptain());
-            } catch (MessagingException e) {
-                // TODO: Fix mailservice exception handling
```
Comment (TimvdLippe) [inline src/main/java/ch/wisv/areafiftylan/service/TeamServiceImpl.java]: :fire: :fire: :fire: :fire: :fire:

File: src/main/java/ch/wisv/areafiftylan/service/MailServiceImpl.java (hunk 1 above)
Comment (martijn9612) [inline src/main/java/ch/wisv/areafiftylan/service/MailServiceImpl.java]: This `<a>` tag is self-closing.

File: src/main/resources/banner.txt [hunk 3]
```diff
@@ -1,8 +1,9 @@
...
+${AnsiColor.BRIGHT_YELLOW}#       ##############      ${AnsiColor.BRIGHT_BLUE} / /_\ \_ __ ___  __ _  | |_   _| |_| |_ _   _ ${AnsiColor.BRIGHT_YELLOW} | |    / /_\ \|  \| |
+${AnsiColor.BRIGHT_YELLOW}#     ####  ######  ####    ${AnsiColor.BRIGHT_BLUE} |  _  | '__/ _ \/ _` | |  _| | |  _| __| | | |${AnsiColor.BRIGHT_YELLOW} | |    |  _  || . ` |
+${AnsiColor.BRIGHT_YELLOW}#   ######################  ${AnsiColor.BRIGHT_BLUE} | | | | | |  __/ (_| | | |   | | | | |_| |_| |${AnsiColor.BRIGHT_YELLOW} | |____| | | || |\  |
+${AnsiColor.BRIGHT_YELLOW}#   ##  ##############  ##  ${AnsiColor.BRIGHT_BLUE} \_| |_/_|  \___|\__,_| \_|   |_|_|  \__|\__, /${AnsiColor.BRIGHT_YELLOW} \_____/\_| |_/\_| \_/
+${AnsiColor.BRIGHT_YELLOW}#   ##  ##          ##  ##  ${AnsiColor.BRIGHT_BLUE}                                         __/ |
+${AnsiColor.BRIGHT_YELLOW}#         ####  ####        ${AnsiColor.BRIGHT_BLUE}                                        |___/
+${AnsiColor.BRIGHT_RED}:: Running Spring Boot ${spring-boot.version} ::${AnsiColor.BLACK}
```
Comment (martijn9612) [inline src/main/resources/banner.txt]: :space_invader: :space_invader: :space_invader: :space_invader: :space_invader: :space_invader: :space_invader: :space_invader: :space_invader: :space_invader: :space_invader: :space_invader:

**Comment by martijn9612** [comment]
LGTM.

**Pull request merged**